settings.gradle

```diff
@@ -6,7 +6,7 @@ pluginManagement {
 }
 
 plugins {
-    id 'io.micronaut.build.shared.settings' version '5.3.15'
+    id 'io.micronaut.build.shared.settings' version '5.4.5'
```

Comment:
`master` is for Micronaut 4, so you can upgrade to the 6.2.x line.

Reply:
This particular PR is for the 3.9.0 release.

Reply:
oh sorry I missed that, I'll shut up :)